src/App.js

```diff
@@ -1,26 +1,10 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] What do you think of making this component a class-based component?. From the requirement of this project, this should be a class-based component and, not a functional component.